jsm.go

```diff
@@ -102,22 +105,45 @@ type apiPagedRequest struct {
 	Offset int `json:"offset"`
 }
 
-// accountStats returns current statistics about the account's JetStream usage.
-type accountStats struct {
-	Memory  uint64 `json:"memory"`
-	Store   uint64 `json:"storage"`
-	Streams int    `json:"streams"`
-	Limits  struct {
-		MaxMemory    int64 `json:"max_memory"`
-		MaxStore     int64 `json:"max_storage"`
-		MaxStreams   int   `json:"max_streams"`
-		MaxConsumers int   `json:"max_consumers"`
-	} `json:"limits"`
+// AccountInfo contains info about the JetStream usage from the current account.
+type AccountInfo struct {
+	Memory  uint64        `json:"memory"`
+	Store   uint64        `json:"storage"`
+	Streams int           `json:"streams"`
+	Limits  AccountLimits `json:"limits"`
+}
+
+// AccountLimits includes the JetStream limits of the current account.
+type AccountLimits struct {
+	MaxMemory    int64 `json:"max_memory"`
+	MaxStore     int64 `json:"max_storage"`
+	MaxStreams   int   `json:"max_streams"`
+	MaxConsumers int   `json:"max_consumers"`
 }
 
 type accountInfoResponse struct {
 	apiResponse
-	accountStats
+	AccountInfo
+}
+
+// AccountInfo retrieves info about the JetStream usage from the current account.
+func (js *js) AccountInfo() (*AccountInfo, error) {
```

Comment:
btw the type that represents this in the server is called `JetStreamAccountStats`
https://github.com/nats-io/nats-server/blob/00db4ba7a1c7520641805ca14e6aae608d37b1d6/server/jetstream.go#L52